lib/screens/admin_order_detail_page.dart

```diff
@@ -0,0 +1,585 @@
+        '?サイズ/なにいろ',
+        style: TextStyle(
+          fontSize: 14,
+          fontWeight: FontWeight.w300,
+          color: paletteBlackColor,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildOrderItemPrice(OrderItem orderItem, Product product) {
+    return Container(
+      child: Text(
+        "¥" + NumberFormat("#,##0").format(orderItem.price),
+        style: TextStyle(
+          fontSize: 16,
+          fontWeight: FontWeight.w600,
+          color: paletteBlackColor,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildOrderItemQuantity(OrderItem orderItem, Product product) {
+    return Container(
+      height: 85,
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: <Widget>[
+          Row(
+            children: <Widget>[
+              Container(
+                color: kPaletteWhite,
+                height: 30,
+                width: 40,
+                alignment: Alignment.center,
+                child: Text(
+                  orderItem.quantity.toString(),
+                  style: TextStyle(
+                    fontSize: 16,
+                    fontWeight: FontWeight.bold,
+                    color: paletteBlackColor,
+                  ),
+                ),
+              ),
+              Text(
+                ' 点',
+                style: TextStyle(
+                  fontSize: 16,
+                  fontWeight: FontWeight.w300,
+                  color: paletteBlackColor,
+                ),
+              )
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildOrderPriceDescriptionBar() {
+    return Container(
+      padding: EdgeInsets.symmetric( horizontal: 16.0 ),
+      height: 40,
+      child: Container(
+        color: kPaletteWhite,
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: <Widget>[
+            Container(
+              child: Text(
+                '総合計',
+                style: TextStyle(
+                  color: paletteBlackColor,
+                  fontSize: 16,
+                  fontWeight: FontWeight.w600,
+                ),
+              ),
+            ),
+            Container(
+              child: Text(
+                "¥" + NumberFormat("#,##0").format( widget.order.getOrderPrice() ),
+                style: TextStyle(
+                  color: paletteDarkRedColor,
+                  fontSize: 16,
+                  fontWeight: FontWeight.w600,
+                ),
+              ),
+            ),
+          ],
+        )
+      ),
+    );
+  }
+
+  Widget _buildBuyerDetails() {
+    return Container(
+      decoration: BoxDecoration(
+        color: kPaletteWhite,
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(40),
+          topRight: Radius.circular(40),
+        ),
+      ),
+      child: Column(
+        children: <Widget>[
+          _buildGreyBar(),
+          _buildBuyerContactInfoBox(),
```

Comment:
maybe rename this to `_buildCustomerContactInfoBox` to be consistent in our naming everywhere?